specfile/prep.py

```diff
@@ -350,25 +350,27 @@ def parse(section: Section) -> "Prep":
                     m.group("o"),
                 )
                 prefix, suffix = line[: m.start("m")], line[m.end("o") :]
-                cls = next(
+                class_ = next(
```

Comment:
I believe that in p-s, we use `klass` instead of `class_` for things like this, I'd suggest to use that for consistency but that's just a nitpick.

Reply:
Ok :slightly_smiling_face: